runtime/doc/windows.txt

```diff
@@ -924,6 +924,26 @@ set in the preview window to be able to recognize it.  The 'winfixheight'
 option is set to have it keep the same height when opening/closing other
 windows.
 
+                                                *preview-popup*
+Alternatively, a popup window can be used by setting the 'previewpopup'
+option.  When set, it overrules the 'previewwindow' and 'previewheight'
+settings.  The option is a comma-separated list of values:
+  `border`, `height`, `width`, `title`, `title_pos`, `footer`, `footer_pos`.
+  see |nvim_open_win()|
+
+Note border only support string values.
+Example: >vim
+        :set previewpopup=border:double,height:10,width:60
+
+A few peculiarities:
+- If the file is in a buffer already, it will be re-used.  This will allow for
+  editing the file while it's visible in the popup window.
+- No ATTENTION dialog will be used, since you can't edit the file in the popup
+  window.  However, if you later open the same buffer in a normal window, you
+  may not notice it's edited elsewhere.  And when then using ":edit" to
+  trigger the ATTENTION and responding "A" for Abort, the preview window will
+  become empty.
```

Comment:
This isn't true for Nvim (it's from [v8.1.1813](https://github.com/vim/vim/commit/2debf1c16b93f8693a785f675320d9e949c96a97)), but maybe this behaviour should be ported given that the popup is `focusable = false`.